### ISSUE TITLE
Highlight project maintainers on community page

### ISSUE THREAD

**Issue by shilman** [storybooks/frontpage#40]
Lets's highlight sub-area maintainers on the Community page.

## Proposal

In https://github.com/storybooks/storybook/pull/6509 I added a config file `automention.yml` that maps github labels to maintainers who are willing to review PRs and provide user support around specific project areas, ranging from view layers such as `react`/`vue`/`angular` to addons like `knobs`/`storysource` to features like `typescript`/`theming`.

Proposal:
- Select a subset of these areas to highlight
- Read latest `automention.yml`
- Display avatars/profile links for the maintainers in each area

**NOTE: needs design** @domyen

**Comment by domyen** [comment]
@shilman how about we update the team page to something like this.
✅ Highlights maintainers who have gone above and beyond to become domain experts
✅ Shows more amazing contributors to give a better sense of how large the community actually is
![image](https://user-images.githubusercontent.com/263385/58740411-2be90c00-83c5-11e9-9667-7230eadaa35e.png)

**Comment by divyanshu-rawat** [comment]
Hello @shilman , @domyen Hope you guys are doing well, I am plannig to contribute to some open source projects so if possible could you guys please let me know if this issue is still open, and if yes can I work on it?

**Comment by divyanshu-rawat** [comment]
@shilman any update on the above issue?

**Comment by domyen** [comment]
@divyanshu-rawat Would love some help implementing this.

**Comment by divyanshu-rawat** [comment]
@domyen , As far as I can see, this is how the current implementation looks like, so my Question is what id the goal of this issue? is it about re-designing the existing design to the one that you have added above.


<img width="1657" alt="Screenshot 2019-11-25 at 09 36 39" src="https://user-images.githubusercontent.com/13464678/69524869-47456600-0f67-11ea-85c1-5e39d48f9d3d.png">

**Comment by domyen** [comment]
Yea that's correct (but not with my face repeated everywhere 😅). For GitHub contributors you can likely reuse some of the work @kylesuss did for [LearnStorybook.com](learnstorybook.com) (scroll down to the bottom).

Will leave the **maintainers** section to be curated by @shilman.

**Comment by divyanshu-rawat** [comment]
@domyen Cool, will start working on this issue, and ping you if I struck. 👍

**Comment by kylesuss** [comment]
Hey @divyanshu-rawat , here is a link to the component that fetches GitHub contributors for your ref:

https://github.com/chromaui/learnstorybook.com/blob/master/src/components/screens/IndexScreen/Community.js

**Comment by divyanshu-rawat** [comment]
Thanks @kylesuss , As far as I can see in the team screen, contributors are hardcoded instead of fetching them from GitHub.


```
const contributors = [
  {
    name: 'Daniel Duan',
    avatarUrl: 'https://avatars1.githubusercontent.com/u/1474548',
  },
  {
    name: 'Kai Röder',
    avatarUrl: 'https://pbs.twimg.com/profile_images/1167896480373362689/CRgdWRVh.jpg',
  },
 ......
];
```

I am planning to start with writing a call to api, to fetch them, for that I will create a draft PR, on the project once I am done with it I will change the status to ready to review. 🚀

**Comment by divyanshu-rawat** [comment]
@domyen  currently TeamItem looks like the below attached img.
<img width="226" alt="Screenshot 2019-12-07 at 12 55 51" src="https://user-images.githubusercontent.com/13464678/70374619-b8d0cd00-18f4-11ea-964c-39bcacadab85.png">


And the UI you added above, the text in it is left-aligned.

<img width="263" alt="Screenshot 2019-12-07 at 13 23 35" src="https://user-images.githubusercontent.com/13464678/70374634-d43bd800-18f4-11ea-8164-65209175d57b.png">

So, in this issue, apart from fetching GitHub contributors, should I also update everything to match the design you posted above.

**Comment by kylesuss** [comment]
@divyanshu-rawat yes the contributors are hard coded on the contributors screen, but the homepage has a dynamic section which fetches them from GitHub. That is what I linked you to.

**Comment by divyanshu-rawat** [comment]
@kylesuss so in summery you mean, I just have to map the fetched contributors from homepage to contributors screen?.

**Comment by kylesuss** [comment]
@divyanshu-rawat I would summarize as:

You can use this exact component from the learnstorybook.com repo (copy & paste it into this repo & rename as appropriate):
https://github.com/chromaui/learnstorybook.com/blob/42931804d166edb991ce2327896087cf4a6562db/src/components/screens/IndexScreen/Community.js#L128-L154

In doing so, you will be able to build the contributors section for this repo:

![Screen Shot 2019-12-10 at 9 28 31 AM](https://user-images.githubusercontent.com/3035355/70548244-75ae6e00-1b2f-11ea-9d31-9352991be0c7.png)

You'll have to add styling and such, but the logic for fetching the contributors & saving them in sessionStorage should be exactly what you need.

**Comment by divyanshu-rawat** [comment]
@kylesuss  Cool, that what I meant thanks :)

**Comment by divyanshu-rawat** [comment]
@kylesuss hey 👋 , Here is the first iteration of PR #78 , feel free to review and share your feedback 👍
